Assets/Scripts/Control&Input/InputManager.cs

```diff
@@ -95,6 +95,11 @@ void OnDestroy()
         RemoveListeners();
     }
 
+    public void SetMouseAndKeyboard()
```

Comment:
And besides, I'm not a fan of having setter-like methods without an actual input parameter. If we keep this method (see my other comment), perhaps rename it to `SetAsMouseAndKeyboard` so that its intent is a little more clear.

Reply:
Yeah same, it is unfortunate that the state has to be updated by an outside class in this case

Reply:
This bit isn't that important so I'll just resolve this comment ¯\_(ツ)_/¯